src/org/etools/j1939_84/bus/j1939/J1939TP.java

```diff
@@ -492,9 +492,6 @@ public void setJ1939(J1939 j1939) {
     }
 
     public void warn(String msg, Object... a) {
-        if (j1939 != null) {
```

Comment:
Since j1939 is only needed in this class to count warnings, we should remove the member and the setter.

Reply:
I don't know how to indicate that this conversation is resolved in a later commit.  I hit the "Resolve conversation" button, but below it still says "Changes requested".

Reply:
I think you can "dismiss" my review and then request a new review.